stored_requests/events/postgres/database_test.go

```diff
@@ -160,24 +174,37 @@ func TestFetchAllErrors(t *testing.T) {
 				AddRow("imp-2", "", "imp").
 				CloseError(errors.New("Some close error.")),
 			wantReturnedError: false,
```

Comment:
Looks like this test case doesn't expect an error back (`wantReturnedError: false`) so how come it expects a recorded error?

Reply:
👍 I updated the code to return the row close error caught in the deferred function. I also got rid of `wantReturnedError` and removed the close error test case because it was very hard to test. You can create a `CloseError` using the DataDog SQL mock but the error only bubbles up from the SQL package and mock if `.Close` is called before `.Next` reaches the end of the row set. I spent some time on this but had some trouble; it's not easy to do given the order in which these methods are called in our code. I'm punting on writing tests for that error case given we have solid test coverage (89%) in this module already. I did manually test that the code behaves properly by forcing execution down the close error code path.